tests/x11/remote_desktop/vino_screensharing_available.pm

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+#
+#
+# Summary: Test if the Screen Sharing GNOME Desktop functionality works
+#   We currently need to install vino package for that
+# Maintainer: Pavel Dostál <pdostal@suse.cz>
+
+use base "x11test";
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    assert_and_click 'menu', 'left', 90;
+    assert_and_click 'settings';
+    assert_screen 'all_settings';
+    type_string 'shar';
+    assert_and_click 'sharing';
+    if (check_screen 'without_screensharing') {
```

Comment:
Please see the recommendation in http://open.qa/api/testapi/#_check_screen . A multi-tag assert_screen is much preferred here, e.g.

```perl
assert_screen [qw(without_screensharing with_screensharing)];
if (match_has_tag 'without_screensharing') {
…
}
elsif (match_has_tag '…') {
…
}
```

Reply:
But now we are back here. Please avoid the check_screen still.
I thought screen sharing is never active by default, why should it be?
In case sharing is already active then we must need to cover both branches still but then else would not be needed because it would be empty

Reply:
It's not active by default but it can be available (currently in SLE) or not (currently in TW).
If it's not available we need to install it. Then we need to check that it really is available.

Reply:
Ok, will you replace the check screen by assert_screen still?